Source/BusinessMomentsAdaptorImage/dataController.ts

```diff
@@ -0,0 +1,33 @@
+import { Request, Response } from 'express';
+import rawData from './dataschema';
+
+export let allData = (req: any, res: any) => {
+    let data = rawData.find((err: any, data: any) => {
```

Comment:
Use `const` instead of `let`. The variable named `data` is never reassigned.

Reply:
Fixed